src/modules/team/controllers/dto/create-team.dto.ts

```diff
@@ -0,0 +1,46 @@
+import { IsString, IsNotEmpty, IsNumber, IsEmail } from 'class-validator';
+
+export default class CreateTeamDto {
+  @IsNotEmpty()
+  @IsString()
+  name: string;
+
+  @IsNotEmpty()
+  @IsString()
+  shortName: string;
+
+  @IsNotEmpty()
+  @IsString()
+  tla: string;
+
+  @IsNotEmpty()
+  @IsString()
+  address: string;
+
+  @IsNotEmpty()
+  @IsString()
+  phone: string;
+
+  @IsNotEmpty()
+  @IsString()
+  website: string;
+
+  @IsNotEmpty()
```

Comment:
If you're specifying a type other than `string` there's no need to add a constraint for not empty, you can remove those